modules/node_modules/@ncigdc/components/GDCAppsDropdown/style.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import { css, arrowIconRule } from 'glamor';
```

Comment:
`arrowIconRule` is not a function provided by glamor, so this will crash 💥 
`insertRule` is what you want here.